enterprise/server/cmd/executor/executor.go

```diff
@@ -297,3 +292,29 @@ func main() {
 	}()
 	env.GetHealthChecker().WaitForGracefulShutdown()
 }
+
+func cleanBuildRoot() {
+	rootDir := runner.GetBuildRoot()
+	entries, err := os.ReadDir(rootDir)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return
+		}
+		log.Warningf("Failed to read build root dir: %s", err)
+		return
+	}
+
+	for _, entry := range entries {
+		// TODO: move cached firecracker/ociruntime images to a different
+		// directory.
+		if entry.Name() == "executor" && !*deleteImageCacheOnStartup {
```

Comment:
This name is flag-defined, right? Can you make sure it's correct even if the value of the flag is overridden?

Reply:
It's not flag-defined - it's a hard-coded string that we sprinkled in a few different places. I'll leave the code as-is, since we want to move image cache artifacts to filecache eventually anyway